app/controllers/groups_controller.rb

```diff
@@ -294,6 +327,23 @@ def add_members
     end
   end
 
+  def handle_membership_request
+    group = Group.find_by(id: params[:id])
+    guardian.ensure_can_edit!(group)
```

Comment:
If `group` is `nil`, we'll end up raising an `Discourse::InvalidAccess` error here instead of a `Discourse::InvalidParameter`.